user-manuals/facilities/manage-parkings.md

```diff
@@ -0,0 +1,79 @@
+# Parkings
+
+The `Facilities app` within HotWax Commerce offers an efficient solution for overseeing queues or parking spaces designated for orders awaiting fulfillment. These queues serve as digital storage areas for various order types, including those awaiting brokering or pre-orders lacking inventory. Through representing these queues as virtual facilities within the app, users can conveniently establish, oversee, archive, and modify them as necessary. This functionality optimizes workflow by effectively organizing orders and facilitating seamless order management procedures.
+
+## Creating a New Parking
+
+1. Navigate to the parking page in the `Facilities App`.
+2. Click on the `(+)` Add New Parking button at the bottom of the page which will open up a new form for creating new parking.
+3. Enter the name, Internal ID, and description for the new parking.
+4. Click on the `(+)` symbol at the bottom of the form to confirm and save the new parking.
+
+## Archiving Unused Parking
+
+1. Access the `parking` page within the `Facilities app`.
+2. Select the `overflow menu` button of the parking you wish to archive.
+3. Choose the `Archive` option to archive the selected parking.
+4. To unarchive parking, click on the `View Box` icon located at the top right corner of the page. This will display a list of all archived facilities. Click on the `Unarchive` icon next to the name of the parking you wish to unarchive.
+
+## Renaming a Parking
+
+1. In the `Parking page`, click the overflow menu of the parking you want to rename.
+2. Choose the option to `rename` the parking.
+3. Enter the new name for the parking and confirm the changes.
+
+There are some default parkings available when you deploy HotWax Commerce, you can create any new parking of your preference and archive any unused parkings.
+
+## Default Parkings 
+
+**Brokering Queue**
+
+Order brokering is the process of analyzing the order priority and determining the optimal fulfillment location, with proximity and inventory availability.  Brokering Queue serves as a waiting area for orders awaiting processing, ensuring that they are handled efficiently. If multiple brokering engines are scheduled to run at various times throughout the day, Orders placed between each brokering session will wait in the queue until the next scheduled processing time.
```

Comment:
brokering engine is one but brokering runs can be multiple we should write in that way